aws_topology/tests/test_template.py

```diff
@@ -863,20 +863,20 @@ def test_process_kinesis_streams(self):
         self.assertEqual(len(topology), 1)
         self.assertEqual(len(topology[0]["relations"]), 0)
         self.assertEqual(len(topology[0]["components"]), 4)
-        self.assertEqual(topology[0]["components"][0]["Name"], "stream_1")
+        self.assertEqual(topology[0]["components"][0]["data"]["Name"], "stream_1")
```

Comment:
Was the left side just wrong and you're fixing it? If not and we're changing the location of values like name we should make sure the StackPack expects them there